src/st/entity_explosion.h

```diff
@@ -28,12 +23,7 @@ void EntityExplosion(Entity* entity) {
     } else {
         entity->posY.val += entity->velocityY;
 
-#ifdef MISSING_ANIMATE_ENTITY_PROTOTYPE
-        if (!((AnimateEntityMissingProto)AnimateEntity)(
```

Comment:
I thought this would be needed on this PR too - turns out the build will compile just fine without it, and this has not been in use for a bit so can be removed

Then: github.com/Xeeynamo/sotn-decomp/commit/267d6973504f741cbf0f1de580b6624bc672d1fe#diff-ff8ba3c96a3ecaf75e685ba0d5c9d733a6ae1428d1ea8d4b0958da8bd566c47f

Now: https://github.com/search?q=repo%3AXeeynamo%2Fsotn-decomp%20MISSING_ANIMATE_ENTITY_PROTOTYPE&type=code